### ISSUE TITLE
fix android support

### ISSUE THREAD

**Issue by bengreenier** [bengreenier/webrtc-unity-plugin#7]
In order to support android, we need to build the [Android Webrtc codebase](https://chromium.googlesource.com/chromium/src/+/master/docs/android_build_instructions.md) with our necessary changes (to facilitate an IL2cpp compatible interface). In order to do this, the differences from the [webrtc-uwp](https://github.com/webrtc-uwp/webrtc-uwp-sdk) and [webrtc](https://webrtc.googlesource.com/src/+/refs/heads/master/) codebases must be reconciled. Since that work will need to be done in the future (when webrtc-uwp and webrtc decide to merge, assuming they do), we've opted to not do that work now.

__TL;DR: Unity Android/IOS support isn't functional at this time, as there isn't an easy way to apply the uwp changesets that modify the plugin api to the android/ios codebase.__

**Comment by lukasrandom** [comment]
Are there any updates regarding ios and android support?

**Comment by bengreenier** [comment]
No, sorry. Unfortunately it's not likely I'll have time to tackle this anytime soon. However, you might check out https://github.com/Unity-Technologies/com.unity.webrtc - I'm not sure if Android/IOS is on their roadmap, but they're making advances beyond what this repo will support, and is backed by Unity.
